database/migrations/2021_01_11_143707_create_doctors_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateDoctorsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('doctors', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('email')->unique();
+            $table->string('phone')->nullable();
+            $table->string('photo');
+            $table->string('degree_copy');
+            $table->integer('gender');
+            $table->timestamp('activated_at');
```

Comment:
This field id nullable as when he registers for the first time he will not be activated so it will be null